docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md

```diff
@@ -66,82 +65,92 @@ Rancher v2.5 replaced the former catalog system with a new **Apps & Marketplace*
 
-* Rancher tools such as Logging or Monitoring are included under the Rancher label.
-* Partner charts reside under the Partners label.
-* Custom charts will show up under the name of the repository.
-
-All three types are deployed and managed in the same way.
+All three types of charts are deployed and managed in the same way.
 
 :::note
 
-Apps managed by the Cluster Manager (the global view in the legacy Rancher UI) should continue to be managed only by the Cluster Manager, and apps managed with **Apps** in the new UI must be managed only by **Apps**.
+Apps managed by the Cluster Manager (the global view in the legacy Rancher UI) continue to be managed only by the Cluster Manager, and apps managed with **Apps** in the new UI must be managed only by **Apps**.
 
 :::
 
-### Repositories
+To access the **Charts** page:
+
+1. Click **☰**. Under **Explore Cluster** in the left navigation menu, select a cluster.
+1. In the left navigation menu on the **Cluster Dashboard**, click **Apps > Charts**.
+
+## Repositories
 
-From the left sidebar select **Repositories**.
+The **Repositories** page lists your Helm repositories. These include traditional Helm endpoints which have an index.yaml, and Git repositories that are cloned and point to a specific branch. To use custom charts, add your repository here. Once added, you can access custom charts in the **Charts** page, listed under the name of the repository.
 
-These items represent Helm repositories, and can be either traditional Helm endpoints which have an index.yaml, or Git repositories which will be cloned and can point to a specific branch. In order to use custom charts, simply add your repository here and they will become available in the Charts tab under the name of the repository.
+To access the **Repositories** page:
+
+1. Click **☰**. Under **Explore Cluster** in the left navigation menu, select a cluster.
+1. In the left navigation menu on the **Cluster Dashboard**, click **Apps > Repositories**.
 
 #### Add Custom Git Repositories
 
-Click **Create** and select the target, **Git repository containing Helm chart...** to add a custom Git repository that contains your Helm charts or cluster template definitions.
+To add a custom Git repository that contains your Helm charts or cluster template definitions:
 
-You must enter a name and a Git repository URL. The other fields, including the description, are optional. Enter an alternative branch name if you don't want to pull from the default, `main`.
+1. Click **☰**. Under **Explore Cluster** in the left navigation menu, select a cluster.
+1. In the left navigation menu on the **Cluster Dashboard**, click **Apps > Charts**.
+1. Click **Create**.
+1. Select the target, **Git repository containing Helm chart...**.
+1. You must enter a name and a Git repository URL. The other fields, including the description, are optional. Enter an alternative branch name if you don't want to pull from the default branch, `main`.
```

Comment:
```suggestion
1. You must enter a name and a Git repository URL. The other fields, including the description, are optional. Enter an alternative branch name if you don't want to pull from the default branch, `master`.
```

Reply:
After a certain point in time, Github stopped using `master` as the default when you create a new repo and instead uses `main`. I want to say 2020? Older repos may still use `master` and there may be an option to customize what the default branch is named when you create a new repo...maybe just not mention the default branch name?

Reply:
At first I responded as if it were simply a matter of what the default branch name is. However, is this Rancher automatically seeking to pull from a branch named `master` vs one named `main`? If it's the first situation, we probably shouldn't mention the name of the default -- since owners can customize which branch is the default, and Github switched from naming the auto default branch `master` to `main` at some point in the past few years

Reply:
I'd vote for not mentioning it. Visually the UI uses `master` as the field's placeholder and description. But you'll need to verify with the UI team whether the actual behavior is using the GitHub-configured default or `master` is hardcoded.

Reply:
From Slack -

![image](https://github.com/rancher/rancher-docs/assets/19174201/d84afebf-464c-4ef9-bd6a-0fca22a833b1)